lib/ui/settings/resetsettings.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * ResetSettings & ResetComponent
+ *
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Panel, FormGroup, Checkbox, HelpBlock, Button} from 'react-bootstrap';
+
+import InstrumentPanel from '../instrumentpanel';
+import {
+  reloadWithLayout,
+  getLayoutName,
+  } from '../../util/browser';
+import {
+  removeKeysByName,
+  old_layoutKeyName,
+  layoutsKeyName,
+  startConnectedKeyName,
+  preferredUnitsKeyName
+  } from '../../util/localstorage';
+
+
+export default class ResetSettings extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {}
+  }
+
+  render() {
+    return (
+      <ResetComponent instrumentPanel={this.props.instrumentPanel}/>
+    );
+  }
+};
+
+ResetSettings.propTypes = {
+  instrumentPanel: PropTypes.instanceOf(InstrumentPanel).isRequired
+};
+
+export class ResetComponent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleChange = this.handleChange.bind(this);
+    this.doReset = this.doReset.bind(this);
+    this.state = {};
```

Comment:
The checkboxes start unchecked? Should they all be checked by default?

Reply:
No checkbox is checked by default.
It's voluntary to start soft.
It is not necessarily necessary to purge all the layouts if you can get by without them.